master/internal/api_job.go

```diff
@@ -6,14 +6,32 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
+	"github.com/determined-ai/determined/master/internal/grpcutil"
+	"github.com/determined-ai/determined/master/internal/job"
 	"github.com/determined-ai/determined/master/internal/sproto"
 	"github.com/determined-ai/determined/proto/pkg/apiv1"
 	"github.com/determined-ai/determined/proto/pkg/jobv1"
 )
 
+func obfuscateJob(job *jobv1.Job) jobv1.LimitedJob {
```

Comment:
nit: Would this be better in obfuscate.go in `authz`?

Reply:
good idea. we chatted offline and we'll leave this to get added in agent rbac work Corban is adding.